scaleway/docs-content
# feat(highlights): available in MDX

## The what
Adding `Highlights` component inside MDX. Will display highlight cards based on data from `blocks/highlights`.

Related to https://github.com/scaleway/docs/pull/1584

## To test
- Make sur `docs` is on the branch of this [PR](https://github.com/scaleway/docs/pull/1584)
- `yarn clean && yarn build --pages=home,tutorials && yarn serve`
- Go on Homepage: you should see a nice line of Tutorials !
- Make sure Highlights at the top of the Tutorials page is OK
